packages/manager/modules/billing/src/main/billing-main.html

```diff
@@ -1,11 +1,14 @@
-<oui-header heading="{{:: 'billing_main_title' | translate }}">
+<oui-header
+    data-heading="{{:: 'billing_main_title' | translate }}"
+    data-description="{{:: $ctrl.coreConfig.getRegion() !== 'US' ? 'billing_main_description' : '' | translate }}"
```

Comment:
Question: Empty heading for US? is it as per the requirements?

Reply:
It is planned in the future to have US content, but it is not yet adapted and ready.